marklogic-data-hub/src/main/java/com/marklogic/hub/flow/impl/FlowRunnerImpl.java

```diff
@@ -641,34 +641,6 @@ protected void applyArtifactSettings(Step step, Map<String, Object> stepConfig)
                     stepOptions.put("outputFormat", artifactJson.get("targetFormat").asText());
                     stepConfig.put("fileLocations", fileLocations);
                 }
-                ObjectNode artifactSettingsJson = null;
```

Comment:
I am wondering if this code is needed yet. Is it possible for a client to specify any of these inputs as part of artifact settings? Basically, I'm wondering what the schema for a "settings" is, as all of these things can already be specified in multiple other places.

Reply:
It is a combination of this [ticket](https://project.marklogic.com/jira/browse/DHFPROD-3921) for creating the settings and this [ticket](https://project.marklogic.com/jira/browse/DHFPROD-3922) for running an ingestion. We should update the Swagger Docs to contain the settings schema.